src/services/usergroup.service.ts

```diff
@@ -145,3 +145,25 @@ export const axVerifyRequest = async (token) => {
     return { success: false };
   }
 };
+
+export const axCheckUserGroupMember = async (userGroupId, userId) => {
+  try {
+    const { data } = await http.get(`${ENDPOINT.USER}/v1/user/groupMember`, {
+      params: { userGroupId, userId }
+    });
+    return { success: true, data };
+  } catch (e) {
+    console.error(e);
+    return { success: false };
+  }
+};
+
+export const axJoinOpenUserGroup = async (userGroupId) => {
```

Comment:
AFAIK this function is utilized for open and closed both type of request `axJoinOpenUserGroup` please refactor this one to `axJoinUserGroup`